go/src/socialapi/workers/countly/api/api.go

```diff
@@ -0,0 +1,123 @@
+package api
+
+import (
+	"koding/db/mongodb/modelhelper"
+	"koding/tools/utils"
+	"net/http"
+	"net/url"
+	"socialapi/config"
+	"socialapi/models"
+	"socialapi/workers/common/response"
+	"socialapi/workers/countly/client"
```

Comment:
I was just going to suggest "group packages together if they start with koding or socialapi"

somthing like `-group "socialapi,koding"` and output would be like 

```diff
$ goimports -local socialapi -local koding -group "socialapi,koding" -w api.go
$ git diff
diff --git a/go/src/socialapi/workers/countly/api/api.go b/go/src/socialapi/workers/countly/api/api.go
index ed03ffd110..10ca911a78 100644
--- a/go/src/socialapi/workers/countly/api/api.go
+++ b/go/src/socialapi/workers/countly/api/api.go
@@ -1,10 +1,12 @@
 package api

 import (
-       "koding/db/mongodb/modelhelper"
-       "koding/tools/utils"
        "net/http"
        "net/url"
+
+       "koding/db/mongodb/modelhelper"
+       "koding/tools/utils"
        "socialapi/config"
        "socialapi/models"
        "socialapi/workers/common/response"
``` 

may be we could open a PR too, i now some people uses a modified version of goimports just for this reason